tests/certification/pubsub/azure/servicebus/topics/components/authentication/service_bus.yaml

```diff
@@ -4,7 +4,7 @@ metadata:
   name: messagebus
   namespace: default
 spec:
-  type: pubsub.azure.servicebus
+  type: pubsub.azure.servicebus.topics
```

Comment:
should not be necessary due to the alias right?

Reply:
Alias is defined in the runtime, not here in the test